.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -35,8 +35,9 @@ body:
       label: NuGet package version
       description: Specify the version of Windows App SDK (or Project Reunion) you're using.
       options:
-        - "1.2.220727.1-experimental1"
-        - "1.1.4"
+        - "1.2.0-preview1"
```

Comment:
Shouldn't this be [1.2.220902.1-preview1](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.2.220902.1-preview1)?

Reply:
Yeah, we can keep it consistent. Fixed in latest commit.